src/shared/constants/apiEndpoints.ts

```diff
@@ -1 +1,2 @@
-// This folder will contain or hold constants. For example in this particular file we can store the api endpoints as constants
\ No newline at end of file
+// This folder will contain or hold constants. For example in this particular file we can store the api endpoints as constants
+export const CLI_COMMAND_URL="http://localhost:8080/cli"
```

Comment:
Would it be better if this derived via config/env var?

Reply:
Yes. In fact NextJS has built-in support for that, So we can have .env, .env.development, and so on to handle them.